composer.json

```diff
@@ -5,6 +5,7 @@
     "keywords": ["Symfony", "bundle", "jwt", "jws", "authentication", "api", "rest"],
     "homepage": "https://github.com/lexik/LexikJWTAuthenticationBundle",
     "license": "MIT",
+    "minimum-stability": "dev",
```

Comment:
Not sure it's a good idea to change this.

Reply:
It's only to allow me working with Symfony 4.4 and Symfony 5 that have not been released yet. Once done, I'll remove it ;)